docs/how-to-guides/renkulab/external-storage.rst

```diff
@@ -0,0 +1,206 @@
+    :alt: GIAB bucket options configuration
+
+
+- On the last page, pick any name for the storage (e.g. ``Giab``), than mark
+  this as Read-only. You can change the "Mount point" too; that determines the
+  virtual folder where this is mounted in the running sessions. 
+  Click on the "Add storage" button.
+
+  .. image:: ../../_static/images/cloud_storage_4.png
+    :align: center
+    :alt: GIAB bucket final steps
+
+.. note::
+  The "mount point" is relative to the root of the project path in the session.
+
+You can now start a new session and access the ``giab`` bucket at the ``external_storage/giab`` path in your project.
+
+
+Azure Blob
+~~~~~~~~~~
+
+You can opt to add Azure Blob storage to your project. We host some data files in the folder ``test-data``
+hosted on Azure and show how to attach it to a Renku interactive session:
+
+- Navigate to the ``Settings - Cloud storage`` in your project.
+
+- Click on the ``Add Cloud Storage``  button and select ``azureblob`` for the
+  ``Storage Type``. Click on the ``Next`` button at the bottom.
+
+  .. image:: ../../_static/images/cloud_storage_5.png
+    :align: center
+    :alt: Azure blob storage selection
+
+- Fill in the form by filling the ``Source path`` to mount (keep it blank to 
+  mount the default root, or specify the folder) and the Account Name. Fill
+  in the ``Shared Key``.  Click on the ``Next`` button.
+
+  .. image:: ../../_static/images/cloud_storage_6.png
+    :align: center
+    :alt: Azure blob options configuration
+
+- On the last page, pick any name for the storage (e.g: `blob_storage``), then mark
+  this as Read-only. You can change the ``Mount point`` too; that determines the
+  virtual folder where this is mounted in the running sessions.
+  Click on the ``Add storage``  button.
+
+  .. image:: ../../_static/images/cloud_storage_7.png
+    :align: center
+    :alt: Azure blob final steps
+
+-  When you start your session, the blob storage will be available at the specified mount point. 
+
+WebDAV
+~~~~~~
+Many storage services, offer access via the WebDAV protocol. Renku supports
+mounting WebDAV resources in interactive sessions. 
+
+  .. warning:
+      We strongly encourage you to generate a token for accessing your private 
+      WebDAV shares to avoid sharing your account password.
+
+Here we show an example of adding an `ETH Polybox <https://polybox.ethz.ch>`_ drive to
+the session, but the procedure will be similar for any storage accessible via WebDAV. 
+
+- Navigate to ``Settings - Cloud storage`` in your Renku project.
+- Click on the "Add Cloud Storage"  button and select ``webdav`` for 
+  the "Storage Type".  Click on the "Next" button at the bottom.
+
+  .. image:: ../../_static/images/cloud_storage_8.png
+    :align: center
+    :alt: WebDAV storage selection
+
+- Fill in the form by filling the ``Source path`` to mount (keep it blank to mount the 
+  default root, or specify the folder), and the URL to specify the host to connect to.
```

Comment:
since we say this example is for polybox, should we have the URL in the text also instead of just in the screenshot? Then at least our data scientists can copy it directly.